index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <script src="http://cdn.mathjax.org/mathjax/2.3-latest/MathJax.js?config=MML_HTMLorMML"></script>
```

Comment:
Also let me point out that Rails' philosophy (which you may or may not want to follow) is to compile all JS into one file (and same for the CSS), so that it is downloaded in one request (less overhead and stays cached after the first page load).